docs/Language-Definition.md

```diff
@@ -205,460 +205,454 @@ date("2023-08-14") - date("2023-08-13") == duration("24h")
 
 ## Built-in Functions
 
-### all(array, predicate)
-
-Returns **true** if all elements satisfies the [predicate](#predicate).
-If the array is empty, returns **true**.
-
-```expr
-all(tweets, {.Size < 280})
-```
-
-### any(array, predicate)
+### len(v)
 
-Returns **true** if any elements satisfies the [predicate](#predicate).
-If the array is empty, returns **false**.
+Returns the length of an array, a map or a string.
 
-### one(array, predicate)
+### get(v, index)
 
-Returns **true** if _exactly one_ element satisfies the [predicate](#predicate).
-If the array is empty, returns **false**.
+Retrieves the element at the specified index from an array or map `v`. If the index is out of range, returns `nil`.
+Or the key does not exist, returns `nil`.
 
 ```expr
-one(participants, {.Winner})
+get([1, 2, 3], 1) == 2
+get({"name": "John", "age": 30}, "name") == "John"
 ```
 
-### none(array, predicate)
-
-Returns **true** if _all elements does not_ satisfy the [predicate](#predicate).
-If the array is empty, returns **true**.
+#### Type Conversion Functions
```

Comment:
Let's keep  categories on the second level. `##`.

Reply:
If Categories will be at the second level, should we remove the `## Built-in Functions` heading that these are all currently under?

Reply:
Renamed it to "Miscellaneous Functions" as if everything else is an h2, this only has the misc functions in it. And I moved misc to the bottom of the functions list since they are miscellaneous